onboarding/src/main/java/com/hedvig/onboarding/chooseplan/OnboardingAdapter.kt

```diff
@@ -27,24 +19,26 @@ import com.hedvig.app.util.extensions.view.setHapticClickListener
 import com.hedvig.app.util.extensions.view.show
 import com.hedvig.app.util.extensions.viewBinding
 import com.hedvig.app.util.invalidData
+import com.hedvig.onboarding.R
+import com.hedvig.onboarding.chooseplan.ChoosePlanActivity.Companion.COMBO
+import com.hedvig.onboarding.chooseplan.ChoosePlanActivity.Companion.CONTENTS
+import com.hedvig.onboarding.chooseplan.ChoosePlanActivity.Companion.TRAVEL
+import com.hedvig.onboarding.databinding.PlanCardBinding
 
 class OnboardingAdapter(
     val viewModel: ChoosePlanViewModel,
-    val marketProvider: MarketManager
-) :
-    ListAdapter<OnboardingModel, OnboardingAdapter.ViewHolder>(OnboardingDiffUtilCallback()) {
+    private val marketProvider: MarketManager
+) : ListAdapter<OnboardingModel, OnboardingAdapter.ViewHolder>(OnboardingDiffUtilCallback()) {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) = when (viewType) {
         R.layout.plan_card -> ViewHolder.QuoteType(parent)
-        R.layout.generic_error -> ViewHolder.Error(parent)
-        else -> {
-            throw Error("Unreachable")
-        }
+        com.hedvig.app.R.layout.generic_error -> ViewHolder.Error(parent)
+        else -> throw Error("Unreachable")
     }
 
     override fun getItemViewType(position: Int) = when (getItem(position)) {
         is OnboardingModel.Bundle -> R.layout.plan_card
-        OnboardingModel.Error -> R.layout.generic_error
+        OnboardingModel.Error -> com.hedvig.app.R.layout.generic_error
```

Comment:
Is this needed? If not, lets do this to remove all the prefixes:

```suggestion
        OnboardingModel.Error -> R.layout.generic_error
```

Reply:
Yeah that resource is in the app module so we need to specify the whole path